Resolved SC1113 and SC2068 Reported by Shellcheck

ThisNekoGuy opened the pull request:
Shellcheck claims two errors are present with this duckstation launch script; this commit fixes them.

ThisNekoGuy committed:
Resolved SC1113 and SC2068 Reported by Shellcheck

Shellcheck claims two errors are present with this duckstation launch script; this commit fixes them.

loathingKernel commented:
Hi, thanks for the PR. The catch-all argument is intentionally unquoted to allow argument splitting.

ThisNekoGuy committed:
Permit Word Splitting for Arguments to Duckstation

Word splitting was apparently intentional; undid the quotes for it and provided a shellcheck ignore for that particular error message

loathingKernel commented:
`shellcheck` might have been correct the first time, I will have to look into it to understand it better. In any case, merging.

Thank you!
